magentov2/README.md

```diff
@@ -1,8 +1,16 @@
-# indodana paylater payment method module for magento >= 2.3.x 
+# indodana paylater payment method module for magento 2.4.0 >= 2.3.x 
```

Comment:
Is `2.4.0` here intended?

Reply:
you can use version 2.3.x

Reply:
already changes to # indodana paylater payment method module for magento 2.4.0 or magento 2.3.x